pom.xml

```diff
@@ -30,7 +30,7 @@ under the License.
   </parent>
 
   <artifactId>maven-dependency-plugin</artifactId>
-  <version>3.3.1-SNAPSHOT</version>
+  <version>3.3.2-SNAPSHOT</version>
```

Comment:
Why?

Reply:
I updated to version for local testing and committed it by mistake.